aya/src/obj/btf/btf.rs

```diff
@@ -291,6 +293,26 @@ impl Btf {
             type_id: root_type_id,
         })
     }
+
+    #[allow(dead_code)]
```

Comment:
🤔

Reply:
oops, fixed it by making it pub, which was what I intended to do :laughing: